release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -348,6 +348,9 @@ When you create a service that uses multiple IP address families, you must expli
 
 For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2045576[BZ#2045576].
 
+////
+This section was taken out becuase the feature was not included in the base release of 4.10.0.
```

Comment:
Suggest removing the content completely instead of commenting it out. It can easily be put back in a subsequent PR.